ERROR    processes.hathiTrust:hathiTrust.py:84 Unable to read hathifile url badURL